pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -7,28 +7,29 @@
 , Security
 , testers
 , tmux-sessionizer
-}: let
+}:
+let
 
   name = "tmux-sessionizer";
-  # v0.4.1 is not released yet, but v0.4.0 has version discrepancy between Cargo.toml and Cargo.lock and does not build
-  version = "0.4.0-unstable-2024-02-06";
+  version = "v0.4.2";
```

Comment:
There's a v0.4.3 tag (the release workflow failed, so there's no github release for it.)
For v0.4.2, the Cargo.toml and Cargo.lock weren't updated, so I'd expect `passthru.tests.version` to fail.

See also: https://github.com/jrmoulton/tmux-sessionizer/pull/85

Reply:
Ah! That's weird. I just forced pushed some changes, and when I do the following, both build fine:
```
nix-build -A tmux-sessionizer
nix-build -A tmux-sessionizer.passthru.tests
```
Also, it seems like [Cargo.lock](https://github.com/jrmoulton/tmux-sessionizer/blob/v0.4.2/Cargo.lock#L915) and [Cargo.toml](https://github.com/jrmoulton/tmux-sessionizer/blob/v0.4.2/Cargo.toml#L3) *are* in sync if I go to the commit associated with the release. Maybe the tag was moved to a different commit?

Reply:
Hmm, could be. Nevermind then. :+1: